pygmt/tests/baseline/.gitignore

```diff
@@ -0,0 +1 @@
+/test_*.png
```

Comment:
Add `pygmt/tests/baseline/test_*.png` to the root `.gitignore`?

Reply:
Sure, I can move them to the top-level `.gitignore`